bundles/org.openhab.binding.liquidcheck/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="liquidcheck"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="liquidCheckDevice">
+		<label>Liquid Check Device</label>
+		<description></description>
+		<category>Sensor</category>
+
+		<channels>
+			<channel id="content" typeId="content-channel"/>
+			<channel id="level" typeId="level-channel"/>
+			<channel id="fill-indicator" typeId="fill-indicator-channel"/>
+			<channel id="raw-content" typeId="raw-content-channel"/>
+			<channel id="raw-level" typeId="raw-level-channel"/>
+			<channel id="pump-runs" typeId="pump-runs-channel"></channel>
+			<channel id="pump-runtime" typeId="pump-runtime-channel"></channel>
+			<channel id="measure" typeId="measure-channel"/>
+		</channels>
+
+		<properties>
+			<property name="firmware">Firmware</property>
+			<property name="hardware">Hardware</property>
+			<property name="name">Name</property>
+			<property name="manufacturer">Manufacturer</property>
+			<property name="uuid">UUID</property>
+			<property name="securityCode">Security Code</property>
+			<property name="ip">IP</property>
+			<property name="mac">MAC-Address</property>
+			<property name="ssid">SSID</property>
+		</properties>
```

Comment:
You do not need to define the properties here, I assume they are set by the binding at run time.

Reply:
If you want to have the properties present on the thing in any circumstances, you can keep them here but in that case please use empty values.